lib/github_user_search_app_web/live/search.ex

```diff
@@ -2,54 +2,64 @@ defmodule GithubUserSearchAppWeb.Search do
   @moduledoc false
 
   use GithubUserSearchAppWeb, :live_view
+
   alias GithubUserSearchApp.UsersAPI
   alias Phoenix.LiveView.JS
 
   def mount(_params, _session, socket) do
     # {:ok, user} = UsersAPI.fetch_user("octocat")
     user = dummy_user()
     form = to_form(%{"username" => ""})
-    {:ok, assign(socket, user: user, form: form)}
+    {:ok, assign(socket, user: user, form: form, error: false)}
   end
 
   def render(assigns) do
     ~H"""
-    <div class="flex place-content-center">
-      <div class="w-[327px] flex flex-col gap-4">
+    <div class="flex place-content-center font-normal text-sm md:text-base leading-6 md:leading-9">
+      <div class="w-[327px] md:w-[648px] flex flex-col gap-4">
```

Comment:
Avoid using px for width(only use as a last resort) prefer percentages
[https://www.youtube.com/watch?v=N5wpD9Ov_To](url)

Reply:
Noted